test/unit/template_unit_test.rb

```diff
@@ -67,4 +67,10 @@ def test_tags_delete
     Template.tags.delete('fake')
     assert_nil Template.tags['fake']
   end
+
+  def test_tags_can_be_looped_over
+    Template.register_tag('fake', FakeTag)
```

Comment:
Hmm that's true. Looks like [this test](https://github.com/Shopify/liquid/blob/23d2beed41cd331dbe81efc2054178dec34b0d4a/test/unit/block_unit_test.rb#L45-L49) also has the same problem.